### DISCUSSION TITLE
I would like to submit a pull request for the new tag features. Would you be willing to accept it?

### DISCUSSION THREAD

**Discussion by bowencool** [satnaing/astro-paper#280] [Ideas]
- Show larger size if there are more related articles, see [preview](https://blog.bowen.cool/tags)
- RSS subscriptions for tags with a large number of articles, for [example](https://blog.bowen.cool/tags/frontend)

**Comment by eerison** [comment]
I liked the large tag size feature 🚀 

the RSS subscription how do you use it? do your public consuming this api? or do you use it for something else?

thank you

**Reply by bowencool** [reply]
I'm glad you like the large tag size feature. It has been implemented here: https://github.com/satnaing/astro-paper/pull/399
